docs/node/daemon/install.mdx

```diff
@@ -3,18 +3,18 @@ sidebar_position: 1
 sidebar_label: "Installation"
 id: daemon-installation
 title: Installation
-description: How to install the Kwil Daemon tool
+description: How to install the Kwil daemon
 slug: /daemon/installation
 ---
 
-Until the main Kwil DB repository is open source, the `kwild` command line
-application may be downloaded from the Kwil binary release repository.
+The Kwil command line applications may be downloaded from the
+[latest GitHub release](https://github.com/kwilteam/kwil-db/releases/latest).
 
 ## Download the `kwild` Executable
```

Comment:
It's time for `go run github.com/kwilteam/kwil-db/cmd/kwild@v0.7.0 --autogen` since we opensourced the repo,  but we need small changes on go.mod to make this work

Reply:
Ah, yeah, and/or we can update the Installation page with similar `go install` commands.

Reply:
Also, as @Yaiba said, for `rc1`, we need to remove all `replace` directives and tag all dependencies in order: https://github.com/kwilteam/kwil-db/wiki/Kwil-Release-Process-(proposed)#remove-replace-directives